README.md

```diff
@@ -24,7 +24,6 @@ run on every issue and pull request for all our repos.
 
 ```sh
 git clone https://github.com/artsy/peril-settings.git
-cd peril-settings
```

Comment:
I'll think we'll still need this, eh?

Reply:
🤦‍♂ oh you're totally right. no clue what I was thinking there, added it back in 😄

Reply:
argh yeah, I think I stashed changes at some point, thanks for catching